data_pipeline/combine.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+
+const charity = JSON.parse(fs.readFileSync('data_pipeline/charity.json'));
+const hotline = JSON.parse(fs.readFileSync('data_pipeline/crisisline.json'));
+
+const db = {
+  charity,
+  hotline
```

Comment:
Would it be a good idea to leave this collection as `crisisline` like the original dataset?

Reply:
good call. [updated](https://github.com/CodeForBoulder/upswyng/pull/70/commits/a2c1eb564ac7476f7900b6d640d1c46bfd083a43)